xil_carma_cloud/docker-compose.yml

```diff
@@ -180,7 +181,138 @@ services:
       && wait-for-it.sh 172.3.0.2:11311 -- roslaunch ns3_adapter ns3_adapter.launch
       role_id:='carma_1'
       remap_ns:=/hardware_interface/comms"
+#################
+# carma_2 vehicle
```

Comment:
It might be better to comment out all services related to the second CARMA vehicle, as it is only relevant to one test case. Most of the test cases focus on testing a single vehicle